README.md

```diff
@@ -4,6 +4,9 @@ Wrapping Google tagmanager and adding some helpers to him
 * This script appends automatically the googleTagmanager script.
 * It also adds an iframe inside a noscript tag for non-js fallback
 
+### [Documentation](https://getninjas.github.io/tagmanager-wrap/)
+
+
 ### Install
 
 `npm i tagmanager-wrap  --save`
```

Comment:
Can we add yarn in the documentation?

Even because we appear on his site and there are only the instructions for `npm`, I think it's worth adding` yarn i tagmanager-wrap --save`.

https://yarnpkg.com/en/package/tagmanager-wrap

Reply:
@williamcosta Good point, will do 👍